syntax = "proto3";

package proto;

service FlightsAndPrices {
    rpc SendId(IdMessage) returns (IdResponse);
    rpc RegisterFlight(NewFlight) returns (RegisterResponse);
}

message IdMessage {
    string id = 1;
}

message IdResponse {
    bool isOk = 1;
}

message NewFlight {
    string id = 1;
    string date = 2;
    string departureAirport = 3;
    string arrivalAirport = 4;
    string departureTime = 5;
    string arrivalTime = 6;
    string airline = 7;
    string price = 8;
    int32 seats = 9;
}

message RegisterResponse {
    bool isOk = 1;
}
